helm/harbor-scanner-aqua/templates/deployment.yaml

```diff
@@ -16,14 +16,29 @@ spec:
         app.kubernetes.io/name: {{ include "harbor-scanner-aqua.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
     spec:
+      imagePullSecrets:
+        - name: {{ include "harbor-scanner-aqua.fullname" . }}-registry
+      securityContext:
+        fsGroup: 1000
+        runAsUser: 1000
+        runAsNonRoot: true
+      initContainers:
+        - name: scannercli
+          image: {{ template "harbor-scanner-aqua.scannerImageRef" . }}
+          command:
+            - cp
+          args:
+            - "/opt/aquasec/scannercli"
+            - "/downloads/scannercli"
+          volumeMounts:
+            - name: scannercli
+              mountPath: /downloads
       containers:
         - name: main
-          image: {{ template "harbor-scanner-aqua.imageRef" . }}
-          imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          image: {{ template "harbor-scanner-aqua.adapterImageRef" . }}
+          imagePullPolicy: {{ .Values.scanner.image.pullPolicy | quote }}
           securityContext:
             privileged: false
```

Comment:
could we move `privilege: false` to the `securityContext` for the whole pod? Maybe also `readOnlyFileSystem`?

Reply:
Nope. Some properties of securityContext, such as `privileged` o `readOnlyFileSystem` are only allowed at a container level.